calico-cloud/operations/cluster-management.mdx

```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
 
 To connect new clusters to {{prodname}}, select Managed Clusters in the left nav, and click **Connect Cluster**.
 
-## Recover a deleted managed cluster
+## Delete a managed cluster
```

Comment:
I think this title is misleading. We're not deleting a cluster so much as removing it from the list. 

I suggest we go with something like

`## Remove a cluster from the managed clusters list`

Reply:
Sounds good, updated https://github.com/tigera/docs/pull/961/commits/ccd5b72a48cbbaa8a3a18f47c1787fee68a5779e